assets/charts/control-plane/kube-apiserver/templates/_helpers.tpl

```diff
@@ -0,0 +1,90 @@
+{{ define "containers" }}
```

Comment:
This change could be done in separate PR, as it is not 1.19 update related. This would make reviewing and testing easier.